_dashboards/index.md

```diff
@@ -13,7 +13,9 @@ redirect_from:
 
 # What is OpenSearch Dashboards?
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is the preferred visualization platform for analyzing data in OpenSearch. It serves as the user interface for many openSearch features, including security, alerting, index state management, SQL, and more.
+
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and OpenSearch provides a full suite of plugins that you can use to monitor events, identify trends, and automate recurring activities.
 
 ![Sample dashboards]({site.url}}{{site.baseurl}}/images/dashboards-example.png)
```

Comment:
Missed a curly bracket at the beginning of `site.url` - probably just a copy-paste error but make sure to add it otherwise the link won't display correctly.

This looks like it's the original page though but still needs fixed before publishing. Does this link work when you try to build it locally?